doc/user-guide/faq.md

```diff
@@ -0,0 +1,76 @@
+
+This chapter describes the problems that occur on the creating VS step.
+
+### A virtual schema does not recognise a property and throws the dialect does not support property exception.
+
+```
+VM error:
+com.exasol.adapter.dialects.PropertyValidationException: The dialect <DIALECT_NAME> does not support <PROPERTY_NAME> property. Please, do not set the BIGQUERY_ENABLE_IMPORT property.
+```
+
+**Solutions**:
+
+- Check if the dialect supports the property you are trying to use. Each dialect has its own set of supported properties. Check the [dialect documentation][dialects].
+- Check that you use a Virtual Schema version that supports the specified property. Check the release logs on the [GitHub][dialects] to find out in which version we added the property you need.
+- Check that you don't have typos in the specified property.
+
+### SQLException: No suitable driver found...
+
+**Solutions**:
+
+- Check the JDBC driver you register in the EXAoperation and make you have registered the driver correctly.
+- Check if you have typos in the main class definition or prefix.
+- Now check the driver itself: open the JAR archive and make sure the file `META-INF/services/java.sql.Driver` exists.
+- If the file exists, open it and make sure it contains the driver's main class reference you specified in the EXAoperation.
+- If the file does not exist or does not contain the correct main class reference, you can add it and re-upload the fixed JAR archive. You should also report the problem to the developers of the driver.
+
+### VM error: End of %scriptclass statement not found
+
+**Solutions**:
+
+- If you are using DbVisualizer, add an empty comment `--/` before the create adapter script statement:
+
+```
+--/
+CREATE OR REPLACE JAVA ADAPTER SCRIPT ...
+```
+
+### I have started a `CREATE VIRTUAL SCHEMA` statement, but it is running endlessly without giving any output or an error.
+
+**Solutions**:
+
+- Check if you specified a property `SCHEMA_NAME`. If you do not add this property, the virtual schema will try to read metadata of all tables existing in a source database. It can take very long time.
+- Check how many tables do exist in the schema you have specified. If there are more than a few hundreds of tables, creation of a virtual schema can also take time.
+
+### Can I exclude multiple tables from a virtual schema?
+
+**Answer**: Yes, you can use TABLE_FILTER = 'TABLE1','TABLE2',...
+
+## Selecting From Virtual Schemas
+
+### The virtual schema was created successfully, but when you try to run a SELECT query, you get an `access denied` error with some permission name.
+
+For example:
+
+``` 
+JDBC-Client-Error: Failed loading driver 'com.mysql.jdbc.Driver': null, access denied ("java.lang.RuntimePermission" "setContextClassLoader")
+```
+
+**Solution**:
+
+- This happens because the JDBC driver requires more permissions that we provide by default. You can disable a security manager of the corresponding driver in EXAoperation to solve this problem.
```

Comment:
```suggestion
- This happens because the JDBC driver requires more permissions than we provide by default. You can disable a security manager of the corresponding driver in EXAoperation to solve this problem.
```

Reply:
Add link to EXAop docu.